lib/MusicBrainz/Server/Validation.pm

```diff
@@ -310,6 +311,24 @@ sub is_valid_partial_date
     return 1;
 }
 
+sub is_valid_edit_note
+{
+    my $edit_note = shift;
+
+    # An edit note with only spaces and / or punctuation is useless
+    if ($edit_note =~ /^[[:space:][:punct:]]+$/) {
+        return 0
```

Comment:
Nitpick: a lack of semicolons after the returns, though in-lining them like `return 0 if ...` seems more appropriate anyway.

Reply:
Seems reasonable. Done!